database/seeders/20221108122917-create-users.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const { faker } = require('@faker-js/faker');
+
+module.exports = {
+  up: async (queryInterface) => {
+    const users = generateUsers();
+    await queryInterface.bulkInsert('users', users, {});
+  },
+
+  down: async (queryInterface) => {
+    await queryInterface.bulkDelete('users', null, {});
+  },
+};
+
+function generateUsers() {
+  const users = [];
+  for (let i = 1; i <= 10; i++) {
```

Comment:
Hay que crear 10 usuarios normales y 10 administradores. Tendrían que ser dos for hasta 10 y agregarle el roleId: 1

Reply:
El roleId solo para los administradores, a los usuarios normales ya se les asigna por defecto el rol de usuario